src/renderers/shared/utils/__tests__/ReactErrorUtils-test.js

```diff
@@ -71,5 +72,15 @@ describe('ReactErrorUtils', () => {
       __DEV__ = true;
       global.process = oldProcess;
     });
+
+    it('shouldn\'t call the callback again while executing synchronously induced other call in development', () => {
```

Comment:
Sure thing, should the new one be located in the same file or should I move it somewhere else? gonna prepare better test later 2day